packages/cli/src/lib/implementation/core-config.middleware.integration.test.ts

```diff
@@ -35,3 +51,73 @@ describe('coreConfigMiddleware', () => {
     ).rejects.toThrow(/Provided path .* is not valid./);
   });
 });
+
+const cliWithConfigOptionsAndMiddleware = (
+  cliObj: Partial<GeneralCliOptions>,
+) =>
+  yargsCli<CoreConfig>(objectToCliArgs(cliObj), {
+    options: {
+      ...yargsCoreConfigOptionsDefinition(),
+    },
+    middlewares: [{ middlewareFunction: coreConfigMiddleware }],
+  });
+
+describe('cliWithConfigOptionsAndMiddleware', () => {
```

Comment:
These tests concern both `yargs` CLI and `middleware`. Therefore these tests should live elsewhere, e.g. in a file `cli.<scope>-test.ts` as this file is only supposed to test the middleware function.

Additionally, the `describe` name should either name an existing production code function or a simple description of what scope is being tested, not a test-only created helper function.

Reply:
I will move the tests after in the future. Is this ok?